sites/fast-website/src/app/components/color-swatch/color-swatch.styles.ts

```diff
@@ -0,0 +1,134 @@
+import { css } from "@microsoft/fast-element";
+import { display, heightNumber, neutralFillInputHoverBehavior, neutralFillInputRestBehavior, neutralOutlineHoverBehavior, neutralOutlineRestBehavior, forcedColorsStylesheetBehavior, SystemColors, focusVisible, disabledCursor } from "@microsoft/fast-components";
```

Comment:
strange that prettier isn't catching this long line, can you indent them like the import below?